packages/block-library/src/post-comment-content/edit.js

```diff
@@ -15,13 +24,23 @@ export default function Edit( { context: { commentId } } ) {
 		'content',
 		commentId
 	);
+
+	// Show a placeholder when there is no context.
+	if ( ! commentId ) {
```

Comment:
If no `commentId` exists the message should be something like `no comment exists`.
If `content` is `undefined` has two cases: 
1. still loading
2. comment doesn't exist (this can be handled in a separate PR though and has to do with `useEntityProp`)

Reply:
> If no `commentId` exists the message should be something like `no comment exists`.

Maybe (or probably :sweat_smile:) I'm wrong, but from what @jameskoster wrote in [this comment](https://github.com/WordPress/gutenberg/pull/35183#issuecomment-929304277), I understood that, if you don't have `commentId` defined, it could be one of these two cases:

1. You haven't defined `commentId` in the parent block (i.e., the Comment block). In that case, I assumed that the block responsible for showing a message like `Input the comment ID` would be the Comment block, and the Comment Content block wouldn't be rendered.
2. You are editing a template / template part / etc. that would be rendered inside a Comment block. For this one, I thought you would see a generic message like `The content of the comment`.

Reply:
> If `content` is `undefined` has two cases:
>
> 1. still loading
> 2. comment doesn't exist (this can be handled in a separate PR though and has to do with `useEntityProp`)

I see. Is there a way to know if an entity is still loading or if it doesn't exist after calling `useEntityProp`? I mean, like a value or a prop returned by that hook? That would be useful. :slightly_smiling_face:

Reply:
I guess we could have a placeholder chip [like here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-featured-image/edit.js#L143) when no `commentId` is provided.

>You mean that, if content is not defined, it's not that the comment has no content but that it doesn't exist,

Correct.

Probably there is a couple of things we could do for this first version, where we do not have the comment content editable - also not sure if we need to..
1. Skip the check/placeholder for a comment that exists and has no content
2. Instead of `useEntityProp`, use the `core/data` selectors like:
```
const { comment, isLoading } = useSelect(
	( select ) => {
		const { getEntityRecord, hasFinishedResolution } = select( coreStore );
		const queryArgs = [ 'root', 'comment', commentId ];
		return {
			comment: getEntityRecord( ...queryArgs ),
			isLoading: ! hasFinishedResolution( 'getEntityRecord', queryArgs ),
		};
	},
	[ commentId ]
);
```